kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/MetadataChecksumGenerator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.nio.ByteBuffer;
+import java.util.zip.CRC32;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+
+public class MetadataChecksumGenerator {
+
+    private MetadataChecksumGenerator() {
+    }
+
+    public static String checksumFor(HasMetadata... metadataSources) {
```

Comment:
I considered making this an instance method but managing the life cycle of the CRC and the generator felt odd so went with a static method. Interested to hear if anyone has better ideas.